.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "Launch",
+            "skipFiles": [
+                "<node_internals>/**"
+            ],
+            "autoAttachChildProcesses": true,
+            "console": "integratedTerminal",
+            "program": "${workspaceFolder}/index.js"
+        }
+    ]
+}
```

Comment:
I'm not sure this is needed for the change request. Is it possible that you remove it from your pr ?

Reply:
But I wanted to be able to just press F5 to run the bot ;-;

Point taken though, its not needed here I guess